index.md

```diff
@@ -54,12 +54,19 @@ This satisfies the [team project requirement](https://cs.illinois.edu/academics/
 See the [project gallery](/not-ready) for past student projects.
 
 ## Exams ##
-This class will have two midterm exams and no final exam. The second midterm will be just before finals. You are responsible for all material covered in lectures and assignments.
+This class will have three midterm exams and no final exam. The second midterm will be just before finals. Each exam is 50 minutes long and will be held in the CBTF. You will need to make a reservation in the [CBTF](https://cbtf.engr.illinois.edu) to take the exam.
```

Comment:
"The second midterm will be just before finals" should that be third?

Reply:
Yeah...third